### ISSUE TITLE
Doesn’t seem to work for bookdown-generated pages

### ISSUE THREAD

**Issue by ndarville** [kenaniah/chrome-navigation-plugin#6]
Example: <http://r4ds.had.co.nz/index.html>.

```html
<link rel="next" href="introduction.html">
```

The site has been built with [bookdown](https://github.com/rstudio/bookdown).

Any idea why it doesn’t go to the next page?

**Comment by kenaniah** [comment]
Thanks for the bug report! 

The links are parsed properly, and the other shortcut keys work, so this may be something related to how scroll position is detected. I'm looking into it now.

**Comment by kenaniah** [comment]
Postmortem - bookdown uses a fixed height for `<body>` and wrapping `<div>`s that are scrollable. Checking the body's scroll position was therefore ineffective as the main content area lived inside of a child element. 

Version 9.0 now uses checks the scroll positions of elements recursively starting from the target element when the spacebar is pressed, which allows for a better experience in fixed-height layouts.

**Comment by ndarville** [comment]
Nice one, cheers. Should speed up my R studies significantly. :)

**Comment by ndarville** [comment]
Uh-oh, @kenaniah, looks like this update breaks the feature on sites like <https://forums.somethingawful.com> now. :/

**Comment by kenaniah** [comment]
Indeed. Apparently this patch breaks other websites such as Python docs. Reverting for now until I can find a more sustainable solution to this.
